src/gameplay.js

```diff
@@ -116,21 +123,27 @@ const Gameplay = () => {
     useInterval(() => gameLoop(), speed);
 
     return (
-    
-        <div role="button" tabIndex="0" onKeyDown={e => moveSnake(e)}>
-            <div className="rectangle"/>
-            <div className="score-text"> SCORE: {score}</div>
-            <div className = "name-text"> Welcome, {settings.playerName}</div>
-            <canvas
-                style={{backgroundColor: "lightgreen", position: "absolute", top: "80px"}}
-                ref={canvasRef}
-                width={`${CANVAS_SIZE[0]}px`}
-                height={`${CANVAS_SIZE[1]}px`}
-            />
-            {gameOver &&
-            <dif className="game-over">GAME OVER!</dif>}
-            <div className="startgame-button">
-                <button onClick={startGame}>Start Playing</button>
+        <div>
+            <Link to="/">
+                <FaArrowCircleLeft className="home-button" />
+            </Link>
+            <div className="gameplay-div">
+                <div role="button" tabIndex="0" className="game-screen" onKeyDown={e => moveSnake(e)}>
+                    <div className="rectangle"/>
+                    <div className="score-text"> SCORE: {score}</div>
+                    <div className="name-text"> Welcome, {settings.playerName}</div>
+                    <canvas
+                        style={{backgroundColor: "lightgreen", position: "absolute", top: "180px"}}
+                        ref={canvasRef}
+                        width={`${CANVAS_SIZE[0]}px`}
+                        height={`${CANVAS_SIZE[1]}px`}
```

Comment:
I would recommend moving inline CSS elements to app.css

Reply:
It looks like to me that the Canvas HTML element ignores the class imported from the external CSS style sheet, so I have to keep the inline CSS style for now